### PR TITLE
Issue #13693: use property macros in few coding modules

### PR DESCRIPTION
**Pull request by romani**
Issue #13693

codign package files from top util defaultcomeslast

### CODE REVIEW

File: src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml [hunk 1]
```diff
@@ -45,10 +45,10 @@
  &lt;/pre&gt;</description>
          <properties>
             <property default-value="false" name="ignoreConstructors" type="boolean">
-               <description>control whether to ignore constructors.</description>
+               <description>Control whether to ignore constructors.</description>
             </property>
             <property default-value="false" name="ignoreModifiers" type="boolean">
-               <description>control whether to ignore modifiers (fields, ...).</description>
+               <description>Control whether to ignore modifiers (fields, ...).</description>
```
Comment (nrmancuso) [inline src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml:51]: If we are making improvements,  we need to reword this description, it is nonsense. `Configures check to ignore modifiers on target tokens when considering declaration order.` or something is much better.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml (hunk 1 above)
Comment (romani) [reply src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml:51]: Technically we are not improving. 
We just migrate to macros as it, and this is just minor side effect.

I can do one more commit to change wording. As you suggested.

Comment (nrmancuso) [reply src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml:51]: Ok, I did not realize change in capitalization was from generation, we don’t need to do this.

Comment (romani) [reply src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/DeclarationOrderCheck.xml:51]: please proceed with review.

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**